crates/saya/core/src/lib.rs

```diff
@@ -190,7 +191,7 @@ impl Saya {
 
         let mut state_updates_and_exec_info = vec![];
 
-        let (state_updates, da_state_updates): (Vec<_>, Vec<_>) = future::try_join_all(
+        let (state_updates, _da_state_updates): (Vec<_>, Vec<_>) = future::try_join_all(
```

Comment:
```suggestion
        let (state_updates): (Vec<_>, Vec<_>) = future::try_join_all(
```
Could be removed if not used anymore. 👍 That's way better as you did, having the squashed DA with the proof instead.